crates/hyperdrive-math/src/long/max.rs

```diff
@@ -326,11 +323,8 @@ impl State {
         &self,
         estimate_price: FixedPoint<U256>,
         spot_price: FixedPoint<U256>,
-        checkpoint_exposure: I256,
     ) -> Result<FixedPoint<U256>> {
-        let checkpoint_exposure = FixedPoint::try_from(-checkpoint_exposure.min(int256!(0)))?;
-        let mut estimate =
-            self.calculate_solvency()? + checkpoint_exposure / self.vault_share_price();
+        let mut estimate = self.calculate_solvency()? / self.vault_share_price();
```

Comment:
please verify this -- the long exposure should be accounted for elsewhere, so we don't add it here.